Makefile

```diff
@@ -65,6 +65,7 @@ endif
 ## Ensures NPM dependencies are installed without having to run this all the time.
 webapp/.npminstall:
 ifneq ($(HAS_WEBAPP),)
+	git config --global url."ssh://git@".insteadOf git://
```

Comment:
This is a little weird of a hack AFAIC; I'd really prefer that we cache (or pre-cache) the dependencies and do not spend 10 minutes plus every single CI run re-fetching the same things.

Same for golang, BTW, the imports are taking way too long.

Reply:
@levb I agree that proper caching is the right solution, but in your opinion is the current approach in this PR better than using the `--verbose` flag? It may be best to be consistent and use the `--verbose` flag, but I sided for this because it makes the download take less time.

I've made a ticket to properly handle caching of the dependencies https://mattermost.atlassian.net/browse/MM-45779